### PR TITLE
[Feature] Support AnimalKingdom dataset

### PR DESCRIPTION
**Pull request by Dominic23331**
Add AnimalKingdom dataset

### CODE REVIEW

**Comment by Ben-Louis** [comment]
Thank you so much for your valuable contribution! By the way, have you tried to train any model on this dataset?

**Comment by Dominic23331** [comment]
Not yet. I'll try it in a few days。

**Comment by Ben-Louis** [comment]
> Not yet. I'll try it in a few days。

Thank you. We shall anticipate the inclusion of the model configuration files. Moreover, it is suggested to incorporate a concise overview of this dataset, together with the file hierarchy, in the `docs/en/dataset_zoo/2d_animal_keypoint.md`.

**Comment by Dominic23331** [comment]
I trained AK_ P1 and AK_ P2 two datasets, but the effect is not very good, which is more than three points less than the results in the paper

**Comment by Ben-Louis** [comment]
> I trained AK_ P1 and AK_ P2 two datasets, but the effect is not very good, which is more than three points less than the results in the paper

Would you mind sharing your config and training log? That way, we can help figure out what's going wrong.

**Comment by Dominic23331** [comment]
> > I trained AK_ P1 and AK_ P2 two datasets, but the effect is not very good, which is more than three points less than the results in the paper
> 
> Would you mind sharing your config and training log? That way, we can help figure out what's going wrong.

Okay, I will upload the config file as soon as possible in the next two days

**Comment by Dominic23331** [comment]
I upload config files and added explanatory documents

**Comment by Ben-Louis** [comment]
> I upload config files and added explanatory documents

Thanks. Have you trained models with any of them in order to verify their accuracy?

**Comment by Dominic23331** [comment]
> > I upload config files and added explanatory documents
> 
> Thanks. Have you trained models with any of them in order to verify their accuracy?

I have tried to train several classes, but the results are not very satisfactory. I will consider how to improve it.

**Comment by Dominic23331** [comment]
Due to the busy work ahead, I apologize for the delay in this PR. During testing, I found that 210 epochs were not enough to fit the training set, so I increased the epochs to 300.

**Comment by Dominic23331** [comment]
This is the result of my training
                    paper    mmpose
ALL              66.06     62.72
leave-k-out 39.3       37.74
mammals    61.59     57.56
amphibians 56.74     53.56
reptiles        56.06     50
birds            77.35     76.79
fishes           68.25     64.3
This result is not ideal

**Review by Tau-J — APPROVED** [review]
